examples/simple/main.tf

```diff
@@ -0,0 +1,10 @@
+module "ecr_ci" {
+  source = "../../"
+
+  ecr_repo = "${aws_ecr_repository.main.name}"
```

Comment:
```suggestion
  ecr_repo = aws_ecr_repository.main.name
```

Reply:
I resolved this all in 1 pr so I wouldn't have 3 commits idk if that was the "right" way but that's what I did